tutorials/python_interfaces.md

```diff
@@ -81,3 +81,66 @@ iterations  1000
 post_iterations  1000
 pre_iterations  1000
 ```
+
+# Gazebo Systems written in Python
+
+Gazebo also provides a way to write systems in Python. This is done using the
+`gz::sim::systems::PythonSystemLoader` system which loads a given python module
+specified by its `<module_name>` parameter. The search path for the module
+includes `GZ_SIM_SYSTEM_PLUGIN_PATH` as well as `PYTHONPATH`. The module is
+expected to provide a function called `get_system` that returns an instance of
+a class that implements the various interfaces in `gz::sim::System`.
+
+Example python system:
+
+<!-- TODO(azeey) Allow including python files in doxygen -->
+<!-- \include examples/scripts/python_api/systems/test_system.py -->
+```python
+
```

Comment:
```suggestion
```python
```

Reply:
Done in https://github.com/gazebosim/gz-sim/pull/2081/commits/24c3d52966e4a1514fcc6d0c30e79daf1050e754